ta/arch/arm/user_ta_header.c

```diff
@@ -25,26 +25,23 @@ const char trace_ext_prefix[]  = "TA";
 /* exprted to user_ta_header.c, built within TA */
 struct utee_params;
 
+#ifdef ARM32
+#define _C(name) name##_c
```

Comment:
This is ugly but a nice hack. Please choose a more descriptive name of the macro since at least I associate `_C()` with some kind of literal constant.

Reply:
Haha :)
How about `_C_FUNCTION(name)`? Not super-descriptive of the intent, but it probably doesn't matter much given how trivial the macro is.

Reply:
Sounds good :-)